content/root/platform-customization.textile

```diff
@@ -101,28 +102,26 @@ In addition, Ably handle all certificate renewals, rolling out to our global end
 
 When using your own custom domain, Ably provide these DNS records for you to CNAME to. Although it is recommended that you implement a similar architecture by using CNAMEs under two different domains (for example, @ably.example.com@ for your primary traffic and @ably.fallback-example.com@ for your fallback traffic, with the latter hosted with a different registrar and using different name servers to the first), this is not mandatory. Therefore, the decision as to whether you need that extra level of redundancy within your own DNS system is yours.
 
-h3. Request a custom environment
+h3. Request a custom endpoint
 
-To enable a custom environment for your enterprise, speak to your customer success manager. They will create the custom environment for you and provide you with a @ClientOptions@ object, containing the following properties:
+To enable a custom endpoint for your enterprise, speak to your customer success manager. They will create the custom endpoint for you and provide you with a @ClientOptions@ object, containing the following options for the @endpoint@ property:
 
-* @environment@: the name of your custom environment. Provide this option when the hostname for realtime (Websocket) and REST (HTTP) connections is the same. Otherwise, specify the following options instead of @environment@:
-** @realtimeHost@: the hostname to use for realtime (WebSocket) connections.
-** @restHost@: the hostname to use for REST (HTTP) connections.
-* @fallbackHostsUseDefault@: whether to use the standard fallback hosts when an error requires the use of an alternative host. If you have a single host for both realtime and REST connections and specify that hostname using the @environment@ flag, then setting this option to @true@ will automatically determine which fallback hosts to use.
-* @fallbackHosts@ : an array of custom fallback hosts to try if an error requires the use of an alternative host. Ensure that you set these manually if specifying @realtimeHost@ and @restHost@ instead of @environment@.
+* a routing policy:
+  * @endpoint=main@ routes to `main.realtime.ably.net` (or `realtime.ably.net`)
```

Comment:
We do not have an alias for `realtime.ably.net` (I think I commented in the ADR, but I guess that wasn't updated)

```suggestion
  * @endpoint=main@ routes to `main.realtime.ably.net`
```